c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -5,8 +5,12 @@ Version 1.6.6
 =============
 
 1. Now the files `apache/mapcache.xml` and `apache/tiles.conf` will be generated automatically,
-   then you can remove your custom build rules. To disable it set in your Makefile:
-   `TILECLOUD_CHAIN ?= FALSE`.
+   then you can remove your custom build rules.
+
+   If you DON'T use tilecloud-chain and you DON'T have a `tilegeneration/config.yaml` configuration
+   file, you MUST disable it by setting the following parameter in your Makefile:
```

Comment:
Looks good. Not sure it is really needed to use capitalized words, standard case is enought is think.